### ISSUE TITLE
MacOS keyboard navigation does not work (e.g Ctrl-A; Ctrl-E; Ctrl-K)

### ISSUE THREAD

**Issue by pedrorolo** [lapce/lapce#2028]
## Lapce Version

0.2.5

## System information
MacOS 13.1 (22C65)

## Describe the bug
MacOS keyboard navigation does not work (e.g Ctrl-A; Ctrl-E; Ctrl-K)

## Additional information

-
